src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/NullableAttributes.cs

```diff
@@ -125,4 +125,41 @@ sealed class DoesNotReturnIfAttribute : Attribute
         /// <summary>Gets the condition parameter value.</summary>
         public bool ParameterValue { get; }
     }
+
+    /// <summary>Specifies that the method or property will ensure that the listed field and property members have not-null values.</summary>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Property, Inherited = false)]
+#if INTERNAL_NULLABLE_ATTRIBUTES
+    internal
+#else
+    [CLSCompliant(false)]
+    public
```

Comment:
I just note noticed the ref isn't being updated. You'll need to add these to the ref so they can actually be used :-)

Reply:
Ah, thanks!
Added the attributes to ref file and also added some properties since that seems to be the convention for the existing attributes.